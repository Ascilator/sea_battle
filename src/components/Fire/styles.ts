import styled from 'styled-components';

export const StyledFire = styled.div`
  font-size: 30px;
  padding: 10px 20px;
  text-align: center;
  width: 200px;
  margin: 20px auto 0;
  background: #0bda51;
`;
