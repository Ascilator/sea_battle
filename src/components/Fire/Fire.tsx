import React, { FC } from 'react';

import { StyledFire } from './styles';

const Fire: FC = () => <StyledFire>Fire!</StyledFire>;

export { Fire };
