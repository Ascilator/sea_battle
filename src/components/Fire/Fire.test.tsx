import renderer from 'react-test-renderer';
import { Fire } from './Fire';

test('Fire render', () => {
  const fire = renderer.create(<Fire />).toJSON();
  expect(fire).toMatchSnapshot();
});
