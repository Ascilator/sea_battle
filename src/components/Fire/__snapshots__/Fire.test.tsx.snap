// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Fire render 1`] = `
<div
  className="sc-bdvvtL jFfNQC"
>
  Fire!
</div>
`;
